app/workers/follows/send_email_notification_worker.rb

```diff
@@ -0,0 +1,17 @@
+module Follows
+  class SendEmailNotificationWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :mailers, retry: 10
+
+    def perform(follow_id, mailer = NotifyMailer.name)
+      follow = ::Follow.find_by(id: follow_id, followable_type: "User")
+      return unless follow&.followable.present? && follow.followable.receives_follower_notifications?
+
+      return if ::EmailMessage.where(user_id: follow.followable_id).
+        where("sent_at > ?", rand(15..35).hours.ago).
+        where("subject LIKE ?", "%#{::NotifyMailer::NEW_FOLLOWER_SUBJECT}").exists?
```

Comment:
any problem with removing the trailing wildcard search operator after the subject?

Reply:
Since we have a test that checks for the exact string I believe it can be removed